changelog.txt

```diff
@@ -1,3 +1,62 @@
+-------------------------------------------------------------------------------
+Version 0.66 - released June 2022
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* Direct Zoom has been improved and is now enabled by default
+* Nurses can now leave the ward without disturbing her patients
+* Patients with Serious Radiation now glow a beautiful shade of green
+* Patients with Jellyitis now wobble wobble wobble
+* The Hotkeys menu has been overhauled to make it easier to use
+* You can now reset your zoom level by pressing '0'
+* A new fax code cheat has been added!
+* The replace machine prompt will now tell you the new strength before purchase
+  (can be disabled in configuration settings)
+* A machine's dynamic info will tell you the strength a machine can be upgraded
+  to if you replaced it (can be disabled in configuration settings)
+* MacOS users can now install and run CorsixTH anywhere
```

Comment:
The .app isn't really installed (I'm nitpicking there), and perhaps we should elaborate at the end to say something along the lines of "rather than just from /Applications", or similar.

Reply:
All LGTM otherwise, including packager notes